_rules/motion-disabled-de951c.md

```diff
@@ -0,0 +1,188 @@
+---
+id: de951c
+name: Motion can be disabled
+rule_type: atomic
+
+description: |
+  This rule checks that functionality that can be operated by device motion or user motion can be disabled to prevent accidental actuation
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+
+authors:
+  - João Vicente
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document](https://dom.spec.whatwg.org/#concept-document).

```

Comment:
Remark: you can use "short style" refs in markdown, _i.e._ use `[link name][]` in the text and have, at the end of the file, a bunch of lines with `[link name]: link destination 'link description'` That will make the files easier to write and to read (because you only need to write the full URL once).

For example:
```markdown
The rule applies to any [HTML document][].

[HTML document]: https://dom.spec.whatwg.org/#concept-document 'Definition of Document'
```

(and if you need to use another display name for the link than the "link name" (typically for plural), you can use, _e.g._ `[HTML documents][HTML document]`)